### ISSUE TITLE
benchmark_latency.py will hang  when --batchsize=1 and --n=2

### ISSUE THREAD

**Issue by katitizhou** [vllm-project/vllm#1658]
benchmark_latency.py will hang if setting --batchsize=1 and --n=2or4or8.
should max_num_seqs been set as args.batch_size * args.n ？
https://github.com/vllm-project/vllm/blob/main/benchmarks/benchmark_latency.py#L23

**Comment by simon-mo** [comment]
Hmm I'm getting a different error, which model are you using?

```
(base) xmo@simon-dev-l4x2:~/vllm/benchmarks$ python benchmark_latency.py --batch-size=1 --n=2
Namespace(model='facebook/opt-125m', tokenizer=None, quantization=None, tensor_parallel_size=1, input_len=32, output_len=128, batch_size=1, n=2, use_beam_search=False, num_iters=3, trust_remote_code=False, dtype='auto')
Traceback (most recent call last):
  File "/home/xmo/vllm/benchmarks/benchmark_latency.py", line 101, in <module>
    main(args)
  File "/home/xmo/vllm/benchmarks/benchmark_latency.py", line 18, in main
    llm = LLM(
  File "/home/xmo/vllm/vllm/entrypoints/llm.py", line 93, in __init__
    self.llm_engine = LLMEngine.from_engine_args(engine_args)
  File "/home/xmo/vllm/vllm/engine/llm_engine.py", line 225, in from_engine_args
    engine_configs = engine_args.create_engine_configs()
  File "/home/xmo/vllm/vllm/engine/arg_utils.py", line 199, in create_engine_configs
    scheduler_config = SchedulerConfig(self.max_num_batched_tokens,
  File "/home/xmo/vllm/vllm/config.py", line 294, in __init__
    self._verify_args()
  File "/home/xmo/vllm/vllm/config.py", line 298, in _verify_args
    raise ValueError(
ValueError: max_num_batched_tokens (32) is smaller than max_model_len (2048). This effectively limits the maximum sequence length to max_num_batched_tokens and makes vLLM reject longer sequences. Please increase max_num_batched_tokens or decrease max_model_len.
```

**Comment by katitizhou** [comment]
After this patch (https://github.com/vllm-project/vllm/pull/1198),  llm model requires max_num_batched_tokens >max_model_len , so if args.batch_size * args.input_len<max_model_len , you will get this error.  Commenting out this line will fix the problem(https://github.com/vllm-project/vllm/blob/main/benchmarks/benchmark_latency.py#L24).

Here is my test case：
python benchmark_latency.py --model falcon-7b --tokenizer falcon-7b --n 2 --batch-size 1 --trust-remote-code

**Comment by simon-mo** [comment]
Ah you are right, I believe the calculation for `max_num_batched_tokens` need to be updated, as well as `max_model_len` for this benchmark script. However, I think `max_num_seqs` should not change because it is orthogonal to `n` as `n` is for sampling per sequence.

**Comment by libratiger** [comment]
is there any update for this question, when i using the qwen/Qwen-7B-Chat for the latency benchmark, it produce the same error.

**Comment by WoosukKwon** [comment]
Hi @katitizhou @libratiger The bug was fixed in #1839. Thanks for reporting the issue!
